project/unpackaged/myPackage/aura/TimesheetFilter/TimesheetFilterController.js

```diff
@@ -0,0 +1,23 @@
+({
+    filter : function (component, event) {
+        component.set('v.filter', true);
+        var filter = component.get("v.filter");
+      
+        console.log(filter);
+        console.log("enviando evento");
```

Comment:
2 suggestions:
a. Remember that you should remove the "logs" of the files before uploading.
b. The logic of the component, goes into the helper. The controller acts just as a "event handler".